python/src/trezorlib/_internal/emulator.py

```
@@ -381,32 +391,40 @@ def __init__(
         self.disable_animation = disable_animation
         self.main_args = list(main_args)
         self.heap_size = heap_size
 
     def start_tropic_model(self) -> None:
-        if self.tropic_model:
+        if not self.tropic_model:
+            return
+
+        try:
             self.tropic_model.start()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)